gptqmodel/utils/audio.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-FileCopyrightText: 2024-2025 ModelCloud.ai
+# SPDX-FileCopyrightText: 2024-2025 qubitium@modelcloud.ai
+# SPDX-License-Identifier: Apache-2.0
+# Contact: qubitium@modelcloud.ai, x.com/qubitium
+
+import base64
+from io import BytesIO
+
+import audioread
+import av
+import librosa
+import numpy as np
+
+
+SAMPLE_RATE=16000
+def _check_if_video_has_audio(video_path):
+    container = av.open(video_path)
+    audio_streams = [stream for stream in container.streams if stream.type == "audio"]
+    if not audio_streams:
+        return False
+    return True
+
+
+def process_audio_info(conversations: list[dict] | list[list[dict]], use_audio_in_video: bool):
```

Comment:
remove the global var `SAMPLE_RATE=16000` and move into process_audio_info as paramter with 16k default. 

```
def process_audio_info(conversations: list[dict] | list[list[dict]], use_audio_in_video: bool, sample_rate: int = 16000):
```